.pre-commit-config.yaml

```
@@ -79,7 +79,7 @@ repos:
     - --strict
 
 - repo: https://github.com/PyCQA/flake8.git
-  rev: 5.0.4
+  rev: 6.0.0
```

This one also needs to be postponed..
```suggestion
  rev: 5.0.4
```